clients/python/README.md

```diff
@@ -9,28 +9,70 @@ downloads are done as efficiently as possible.
 ```python
 import datetime
 
-from objectstore_client import ClientBuilder, NoOpMetricsBackend, TimeToLive
+import urllib3
+
+from objectstore_client import (
+    Client,
+    NoOpMetricsBackend,
+    TimeToIdle,
+    TimeToLive,
+    Usecase,
+)
 
-client_builder = ClientBuilder(
+# This should be stored in a global variable and reused, in order to reuse the connection
+client = Client(
     "http://localhost:8888",
-    "my_usecase",
-    metrics_backend=NoOpMetricsBackend(),  # optionally, provide your own MetricsBackend implementation
+    # Optionally, bring your own metrics backend to record things like latency, throughput, and payload sizes
+    metrics_backend=NoOpMetricsBackend(),
+    # Optionally, enable distributed traces in Sentry
+    propagate_traces=True,
+    # Optionally, provide kwargs for urllib3.HTTPConnectionPool
+    # If you don't specify these, reasonable defaults are automatically applied
+    connection_kwargs={"timeout": urllib3.Timeout(connect=0.5, read=0.5)},
```

Comment:
_nit_: Do we have a better example than timeout, since timeout is now a top-level kwarg?